ManageWikiSettings.php

```diff
@@ -348,6 +348,17 @@
 		'section' => 'edit',
 		'help' => 'The number of days within which a person must make edits to be considered an "active" user.',
 	],
+	'wgMaxCredits' => [
+		'name' => 'Editing attribution',
+		'from' => 'mediawiki',
+		'restricted' => false,
+		'type' => 'integer',
+		'minint' => -1,
+		'maxint' => 0,
```

Comment:
This should probably be higher

Reply:
See https://www.mediawiki.org/wiki/Special:MyLanguage/Manual:$wgMaxCredits

This allows for all or none but probably and also name probably should be ‘Number of Editors to Attribute in footer’?

Reply:
my mistake, i think the limit should be set to say like, 25?